config/defaults_template.yaml

```diff
@@ -40,6 +40,10 @@ languages:
     buildImage: l.gcr.io/google/bazel:latest
     runImage: "{{ .ImagePrefix }}python:{{ .Version }}"
 
+  - language: python_asyncio
+    buildImage: l.gcr.io/google/bazel:latest
+    runImage: "{{ .ImagePrefix }}python:{{ .Version }}"
+
```

Comment:
I am not sure if we want to arrange the language alphabetically. :) I think Paulo knows more about this.

Reply:
Yes alphabetically please. Thanks.

Reply:
I think it is already alphabetical, or did I miss something?

Reply:
Sorry, I mistakenly think python_Asyncio was in front of python. You are all good.